### PR TITLE
Patch: Updated Ruby to 2.3.1

### PR DESCRIPTION
**Pull request by Seppone**
I had no idea to test this patch. Someone should do this before merging.

### CODE REVIEW

File: manual/ruby/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -30,10 +30,10 @@
   #$checksum64 = '184b18eba6be931d7b03b8d15d6bfad3c9712718'
 
   $rubyFolder = '23'
-  $url = 'http://dl.bintray.com/oneclick/rubyinstaller/rubyinstaller-2.3.0.exe?direct'
-  $checksum = 'f7926dd87bc2f436ea1f407073a01ace38285818578a8764e2c1ec93aaead3c6'
-  $url64 = 'http://dl.bintray.com/oneclick/rubyinstaller/rubyinstaller-2.3.0-x64.exe?direct'
-  $checksum64 = '087a7e3404fc2892f7eae6e8d0a12de419d57a8a4d78bbcf95568cb75f086a99'
+  $url = 'https://bintray.com/oneclick/rubyinstaller/download_file?file_path=rubyinstaller-2.3.1.exe'
```
Comment (ferventcoder) [inline manual/ruby/tools/chocolateyInstall.ps1]: Does the url `?direct` no longer work?

Comment (ferventcoder) [inline manual/ruby/tools/chocolateyInstall.ps1]: Other than that, this looks good.

Comment (Seppone) [inline manual/ruby/tools/chocolateyInstall.ps1]: i changed to `?direct`  urls

**Comment by ferventcoder** [comment]
Thanks!

**Comment by perlun** [comment]
I just did `cinst ruby`. It gave me version 2.3.0. Has this change not been published to the package repo?

**Comment by ferventcoder** [comment]
@perlun likely not yet. I was pulling these changes down to publish and there were other changes for other packages.

Thanks for the reminder

**Comment by ferventcoder** [comment]
I just pushed it now

**Pull request merged**